### PR TITLE
Registration revisions

### PR DESCRIPTION
**Pull request by carbonwheels**

### CODE REVIEW

File: src/Register/Register.js [hunk 1]
```diff
@@ -1,18 +1,298 @@
...
+
+    // fetch(request, {mode: 'no-cors'})
+    // .then(function(response) {
+    //   console.log(response); 
+    // }).catch(function(error) {  
+    //   console.log('Request failed', error)  
+    // });
+
+
+    fetch("http://940.121.12.189:8080/api/registrations/", {
```
Comment (ocalvet) [inline src/Register/Register.js]: This IP looks weird

Comment (ocalvet) [reply src/Register/Register.js]: I think that 9 doesn't go there

**Review by ocalvet — CHANGES_REQUESTED** [review]
When you change the IP try to get it from the environment. create-react-app uses environment variables that start with REACT_APP_ and it also lets you use it locally using a .env file

File: src/Register/Register.js (hunk 1 above)
Comment (carbonwheels) [reply src/Register/Register.js]: fixed ip

**Review by ocalvet — APPROVED** [review]

**Pull request merged**